src/d3a/d3a_core/cli.py

```diff
@@ -80,9 +80,10 @@ def main(log_level):
                   ', '.join(_setup_modules)))
 @click.option('-g', '--settings-file', default=None,
               help="Settings file path")
-@click.option('--slowdown', type=int, default=0,
+@click.option('--slowdown', type=float, default=0,
               help="Slowdown factor [0 - 10,000]. "
-                   "Where 0 means: no slowdown, ticks are simulated as fast as possible; "
+                   "Where 0 means: no slowdown, ticks are simulated as fast as possible;"
+                   "You can give also float value to precisely control the simulation speed"
                    "and 100: ticks are simulated in realtime")
```

Comment:
IMO you dont need the added sentence, but if you want, please only add it to the end.
```suggestion
                   "Where 0 means: no slowdown, ticks are simulated as fast as possible;"
                   "and 100: ticks are simulated in realtime."
                   "You can give also float value to precisely control the simulation speed")
```

Reply:
Changes are done

Reply:
@psurender you deleted one line too much. PLease re-add and 100: "ticks are simulated in realtime"

Reply:
ok